bitnami/zookeeper/values.yaml

```diff
@@ -13,7 +13,7 @@
 image:
   registry: docker.io
   repository: bitnami/zookeeper
-  tag: 3.4.14
+  tag: 3.4.15
```

Comment:
Why are you bumping the image tag?

Reply:
I thought my changes on the image would create a new version, but as it will create a new revision, so I change this back.